lib/ambient_context.ml

```diff
@@ -1,5 +1,4 @@
 module Atomic_ = Ambient_context_atomic.Atomic
-module TLS = Ambient_context_tls.Thread_local
```

Comment:
Not sure if this is accidental or on purpose, why would you want to expose this?